src/toggle/toggle.styles.tsx

```diff
@@ -61,6 +62,15 @@ export const Container = styled.div<ContainerStyleProps>`
         }
     }}
 
+    // Container min width to fit content
+    ${(props) => {
+        if (props.$useContentWidth) {
+            return css`
+                min-width: fit-content;
```

Comment:
`min-width: unset;` which you were using originally might be a better fit here to cater for long strings without breaks

![Screenshot 2024-08-21 at 9 25 52 AM](https://github.com/user-attachments/assets/180518e3-7bf2-42dd-8708-80fce067b126)